test/framework/resources/k8s/manifest/container.go

```diff
@@ -52,7 +52,7 @@ func NewCurlContainer() *Container {
 func NewTestHelperContainer(testImageRegistry string) *Container {
 	return &Container{
 		name:            "test-helper",
-		image:           utils.GetTestImage(testImageRegistry, utils.TestAgentImage),
+		image:           "public.ecr.aws/q1l2n4k8/vpc-cni-temp:agent3",
```

Comment:
This is hardcoded for now, if test/agent code changes look good will revert this and push a new image to ecr